code/modules/mob/dead/new_player/sprite_accessories.dm

```diff
@@ -2089,3 +2089,23 @@
 /datum/sprite_accessory/ipc_chassis/zenghupharmaceuticals
 	name = "Zeng-Hu Pharmaceuticals"
 	limbs_id = "zhpipc"
+
+/datum/sprite_accessory/squid_face/
+	icon = 'icons/mob/mutant_bodyparts.dmi'
+	color_src = MUTCOLORS
+
+/datum/sprite_accessory/squid_face/squidward
+	name = "Squidward"
+	icon_state = "squidward"
+
+/datum/sprite_accessory/squid_face/ilithid

```

Comment:
"illithid" is spelled with two "l"s, not one "l"
(don't forget to update the other mentionings of the word "illithid"!)